index.d.ts

```diff
@@ -19,6 +19,8 @@ export interface CreateClientParams {
     httpsAgent?: any;
     proxy?: AxiosProxyConfig;
     headers?: any;
+    application: string;
```

Comment:
You'll want to add question marks after the names of application and integration, to make these optional.